GoogleForCreators/web-stories-wp
# Dashboard: First half of Travel template animated

## Summary

This PR adds animations to the first 5 pages of the Travel template.

## Relevant Technical Choices



## To-do

- Animate the last 5 pages
- Update our Animation spreadsheet with missing animations required to polish the Travel template.

## User-facing changes

- None.  Animations hidden by `enableAnimation` flag.

## Testing Instructions

1.) Toggle `enableAnimation` to `true` in `Dashboard.php`.
2.) Go to "Explore Templates" and hover over the Travel template to see some animation action.


## Screenshots

<img src="https://user-images.githubusercontent.com/40646372/87076895-346e6180-c1d7-11ea-972d-13c6dd7de68d.gif" height="200">

Page 1:
<img src="https://user-images.githubusercontent.com/40646372/87078265-36392480-c1d9-11ea-9712-896f3b16d922.gif" height="300">

Page 2:
<img src="https://user-images.githubusercontent.com/40646372/87078274-39ccab80-c1d9-11ea-9fa7-a3838732a41b.gif" height="300">

Page 3:
<img src="https://user-images.githubusercontent.com/40646372/87078278-3c2f0580-c1d9-11ea-8a89-23baaf259b07.gif" height="300">

Page 4:
<img src="https://user-images.githubusercontent.com/40646372/87078285-3df8c900-c1d9-11ea-85ee-4d3875b0f889.gif" height="300">

Page 5:
<img src="https://user-images.githubusercontent.com/40646372/87078292-405b2300-c1d9-11ea-8d20-45cbb8710190.gif" height="300">
